tests/pytorch/test_numerics.py

```diff
@@ -534,6 +539,7 @@ def _test_e2e_full_recompute(block, bs, dtype, config, recompute=False):
     return outputs
 
 
+@pytest.mark.skipif(_flash_attn_2_available, reason="FlashAttention v2 is not deterministic.")
```

Comment:
Disabling the test entirely seems drastic since this tests things other than attention. Perhaps we could set `NVTE_FLASH_ATTN=0` when FA 2 is detected and reset it at the end of the test.

Reply:
It doesn't seem that drastic since our tests use FA v1 anyway right? Once we can get the deterministic option we could always remove this

Reply:
Yes, this is only a temporary fix to avoid testing FAv2. We will remove this skipif once FA updates their version, which from our conversations with them should be soon. @timmoon10